azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -74,14 +79,14 @@ public Schedule(int scheduleId, int projectId, String projectName,
     this(scheduleId, projectId, projectName, flowName, status, firstSchedTime,
         DateTimeZone.forID(timezoneId), parsePeriodString(period),
         lastModifyTime, nextExecTime, submitTime, submitUser, executionOptions,
-        slaOptions);
+        slaOptions, null);
   }
 
   public Schedule(int scheduleId, int projectId, String projectName,
       String flowName, String status, long firstSchedTime,
       DateTimeZone timezone, ReadablePeriod period, long lastModifyTime,
       long nextExecTime, long submitTime, String submitUser,
-      ExecutionOptions executionOptions, List<SlaOption> slaOptions) {
+      ExecutionOptions executionOptions, List<SlaOption> slaOptions, String cronExpression) {
```

Comment:
In ideal case no. But it will be pretty easy to miss some APIs (especially plugins in other repo). Moreover, people outside LinkedIn might have taken some dependency on previous version.